LabExT/Wafer/Device.py

```diff
@@ -36,6 +38,10 @@ def __init__(self, identifier, in_p, out_p, typ, parameters=None):
         self._id = identifier
         self._in_position = in_p
         self._out_position = out_p
+
+        self.input_coordinate = ChipCoordinate.from_list(in_p)
+        self.output_coordinate = ChipCoordinate.from_list(out_p)
```

Comment:
Thats a pretty big change. Here I have two questions:
* How does this reflect in the output JSONs? How is an instance of `ChipCoordinate` JSON-serializable?
* Is the "New Experiment Wizard" compatible with this? (we should really also have an automated test for the "New Experiment Wizard" as well...)

Reply:
Ah, you're right, that needs to be changed more! For now,` self._in_position` and `self._out_position` are still supported for JSON outputs and the Experiment Wizard.

Reply:
Let me know if you want to add the changes to this PR or not.

Reply:
I would like to do it in a different PR.